CMakeLists.txt

```diff
@@ -389,8 +389,15 @@ set(OD_SOURCEFILES
     ${SRC}/utils/FrameRateLimiter.cpp
     ${SRC}/utils/Helper.cpp
     ${SRC}/utils/LogManager.cpp
-    ${SRC}/utils/LogManagerFile.cpp
-    ${SRC}/utils/LogManagerOgre.cpp
+    ${SRC}/utils/LogManager.h
+    ${SRC}/utils/LogMessageLevel.h
```

Comment:
Then how can I force CMake to include the headers in the generated vcxproj? I rely on Visual Assist X's "Open File in Solution" dialog to jump between files in a project.